### PR TITLE
Add vmanomaly docs section

### PR DESCRIPTION
**Pull request by dima-vm**
Currently without config example, as I've come up with a new design (less customizable and more opinionated, but which would support multiple queries and multiple models).

### CODE REVIEW

File: docs/vmanomaly.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+---
+sort: 20
```
Comment (hagen1778) [inline docs/vmanomaly.md:2]: It is already taken by other page - see https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/docs/url-examples.md?plain=1#L2

File: docs/vmanomaly.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
+---
```
Comment (hagen1778) [inline docs/vmanomaly.md:1]: it is better to put images next to the vmanomaly markdown file to remain consistent with other docs in the folder.

File: docs/vmanomaly.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+   _(simplest in configuration, recommended for getting starting)_
+
+   Uses Facebook Prophet for forecasting. Anomaly score is computed of how close the actual time
+   series values follow the forecasted values (yhat), and whether it’s within forecasted bounds
+   (_yhat_lower_, _yhat_upper_). Anomaly score reaches 1.0 if the actual data values equal to
+   _yhat_lower_ or _yhat_upper_. Anomaly score is above 1.0 if the actual data values are outside
+   the _yhat_lower_/_yhat_upper_ bounds.
+
+   See Prophet documentation: https://facebook.github.io/prophet/
```
Comment (hagen1778) [inline docs/vmanomaly.md:63]:
```suggestion
   See [Prophet documentation](https://facebook.github.io/prophet/)
```

**Review by hagen1778 — CHANGES_REQUESTED** [review]

File: docs/vmanomaly.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+Then, VMAnomaly pushes the metric to vminsert (under user-configured metric name, 
+optionally preserving labels).
+The writer is also pluggable, users can supply their own writers.
+
+
+## Usage
+Script accepts only one parameter -- config file path:
+
+```sh
+python3 vmanomaly.py config_zscore.yaml
```
Comment (f41gh7) [inline docs/vmanomaly.md:115]: Can we add config file examples? Or some description for config file.

File: docs/vmanomaly.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
+---
+sort: 20
```
Comment (valyala) [inline docs/vmanomaly.md:2]:
```suggestion
sort: 23
```

**Comment by valyala** [comment]
Thanks!

File: docs/vmanomaly.md (hunk 4 above)
Comment (dima-vm) [reply docs/vmanomaly.md:117]: See PR description

**Pull request merged**